lib/components/Shell/Shell.module.scss

```diff
@@ -4,43 +4,30 @@
 // provide some common styling that everyone requires:
 :global {
     .link {
-        color: var(--color-hyperlink-rest);
+        color: var(--color-accent-hover);
 
         &:hover:not(.disabled) {
-            color: var(--color-hyperlink-hover);
+            color: var(--color-accent-selected);

```

Comment:
Can you please add some comments explaining why these choices? This looks different to most other files

Reply:
This where the colors suggested by Arturo for links